src/pages/dashboard/index.page.tsx

```diff
@@ -643,7 +641,36 @@ const Dashboard = (): any => {
   const createOfferUrl = CreateMap[tab];
   const { udbMainDarkBlue, textColor } = colors;
 
-  return [
+  const [isVisible, setIsVisible] = useState(false);
```

Comment:
Maybe you can be more specific here?
```suggestion
  const [isRequestModalVisible, setIsRequestModalVisible] = useState(false);
```